Example/ChatExample/ContentView.swift

```diff
@@ -6,19 +6,38 @@ import SwiftUI
 import ExyteMediaPicker
 
 struct ContentView: View {
+    
+    @State private var isAccent:Bool = true
+    @State private var accentColor = Color(red: 83/256, green: 95/256, blue: 249/256)
```

Comment:
there's a color hex init for convenience, could you please use that one?

Reply:
I was trying to replicate your default blue color, so I switch to just using `Color("messageMyBG", bundle: .current)` instead.